sycl/test/inline-asm/asm_16_empty.cpp

```diff
@@ -0,0 +1,73 @@
+// UNSUPPORTED: cuda
+// REQUIRES: gpu,linux
+// RUN: %clangxx -fsycl %s -DINLINE_ASM -o %t.out
+// RUN: %t.out
+
+#include "include/asmcheck.h"
+#include <CL/sycl.hpp>
+#include <iostream>
+#include <string>
+#include <vector>
+
+constexpr int LIST_SIZE = 1024;
```

Comment:
```suggestion
constexpr int LIST_SIZE = 16;
```
Just to make the test faster.